var/spack/repos/builtin/packages/libgcrypt/package.py

```diff
@@ -32,8 +32,10 @@ class Libgcrypt(AutotoolsPackage):
        key algorithms, large integer functions, random numbers and a lot
        of supporting functions. """
     homepage = "http://www.gnu.org/software/libgcrypt/"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
+    url 	 = "http://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.6.tar.bz2"
```

Comment:
Ah okay, that makes sense.

Just as a suggestion: It might be better to provide a fall back for those packages only and make the ```https``` default for everything. It's not really critical, but in a general way I'm for #httpseverywhere.